AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -2101,6 +2101,7 @@ twitch.tv##.follow-prompt
 twitch.tv###twilight-sticky-footer-root
 av-kappa.com,avokazu.com###bott0m0verlay
 thebaffler.com##.boxzilla-word-factory-subscribe-fullprice
```

Comment:
If a fix requires CSS rule, both rules must be changed to CSS to avoid breakage in conetent blockers
```suggestion
thebaffler.com#$#.boxzilla-word-factory-subscribe-fullprice { overflow: auto !important; padding-right: 0 !important; }
```